File: Programming Basics/Conditional Statements - Exercises/06.WorldSwimmingRecord.cs
using System;

class Program
{
    static void Main(string[] args)
    {
        double worldRecord = double.Parse(Console.ReadLine());
        double distance = double.Parse(Console.ReadLine());
        double time = double.Parse(Console.ReadLine());

        double addedTime = (Math.Floor(distance / 15)) * 12.5;
        double swimmed = distance * time;

        double finalTime = addedTime + swimmed;

        if (finalTime >= worldRecord)
        {
            Console.WriteLine($"No, he failed! He was {finalTime - worldRecord:f2} seconds slower.");
        }
        else
        {
            Console.WriteLine($"Yes, he succeeded! The new world record is {finalTime:f2} seconds.");
        }
    }
}

File: Programming Basics - Python/Conditional Statements - Exercises/06.WorldSwimmingRecord.py
from math import floor

record = float(input())
distance = float(input())
time = float(input())

additional_time = floor((distance / 15)) * 12.5

total = time * distance + additional_time

if total < record:
    print(f'Yes, he succeeded! The new world record is {total:.2f} seconds.')

else:
    print(f'No, he failed! He was {total - record:.2f} seconds slower.')